modules/core/src/smithy4s/http/package.scala

```diff
@@ -28,17 +28,16 @@ package object http {
   final def httpMatch[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]](
```

Comment:
fyi this is only used in tests

Reply:
I think we have a team that uses it internally, that's why it's not been removed 😞

Reply:
Also, same thing : can we retain a String taking version for BC (even if it doesn't quite do the correct thing) ?